web/src/lib/stores/assets.store.ts

```diff
@@ -372,35 +372,31 @@ export class AssetStore {
     this.emit(recalculate);
   }
 
-  removeAssets(ids: string[]) {
-    // TODO: this could probably be more efficient
-    for (const id of ids) {
-      this.removeAsset(id);
-    }
-  }
-
-  removeAsset(id: string) {
-    this.assets = this.assets.filter((asset) => asset.id !== id);
-    delete this.assetToBucket[id];
+  removeAssets = (ids: string[]) => {
```

Comment:
Can we avoid the arrow syntax and update the usage to work accordingly?

Reply:
Forgot about that, I will revert to how it was before

Reply:
Maybe add a handleDelete method in the components that calls the asset store that way you can use the simplified syntax there still.

Reply:
I think the syntax isn't too bad after all, might just leave it like this